docs/changelog.md

```diff
@@ -28,6 +28,13 @@ links to the header, placing them after all other header content.
 * Update the list of empty HTML tags (#1353).
 * Add customizable TOC title class to TOC extension (#1293).
 
+## [3.4.5] -- ????-??-??
```

Comment:
I realize the changelog format is new for everyone here. But my understanding of the spec is that we never specify a version number until the actual release is made. Therefore, this line should be removed. The thinking is that there is no reason to be concerned over what the next version will be. Just add to the changelog as you make changes. Then, when release time comes, we do either a bug-fix, point, or major release depending on what we have accumulated. I'm not entirely sure we want to be that freeform about it.

Of course, I understand that what you were trying to accomplish is perhaps get this bugfix out before the next major release. But to do that we should fork master from the tag for 3.4.4 (to the 3.4 branch) and apply the patch to that. Of course, this would remain to get the patch applied to 3.5 as well.

That said, I am hopeful of getting 3.5 out immediately after Python 3.12 is released. As that is scheduled for October, there shouldn't be much wait. So if your okay waiting for 3.5, then remove this line and combine the item below with the list above.

Reply:
That's fine, I honestly didn't have any real plan for when it got released. I don't have an immediate need for this to be released, others may desire this fix sooner. I'll let you decide when to release it. I update this.